### PR TITLE
New G45W app showcase + server .gif

### PR DESCRIPTION
**Pull request by g45t345rt**
This a new wallet application I'm currently developing for the Dero community. I thought it would be cool to showcase it on the GioUI website.

### CODE REVIEW

**Comment by eliasnaur** [comment]
CC @egonelbre .

File: content/doc/showcase/g45w/3.gif
Comment (egonelbre) [inline content/doc/showcase/g45w/3.gif:1]: 3MB is rather large, let's make sure all images/animations are less than (let's say) 250kb.

But, I'm wondering whether the animations should be there in the first place.

Comment (g45t345rt) [reply content/doc/showcase/g45w/3.gif:1]: Ye this is large :S I can save the video as .mp4 it reduces the size to 800kb. Far from 250kb but a huge improvement. I like the animation as it effectively conveys dynamic interactions. In any case, I can remove it and include more images.

Comment (egonelbre) [reply content/doc/showcase/g45w/3.gif:1]: avif could be better -- alternatively, you can try to shorten or reduce the frame-rate.

Note: I don't think having too many images is great either, 1-3 should be sufficient to get the idea across. If people want to learn more they can navigate to the application site.

**Comment by g45t345rt** [comment]
Replaced gif with another image. Thanks a lot!

**Comment by egonelbre** [comment]
Squashed and merged through sr.ht.

**Pull request closed without merging**